INN/umbrella-caribbean
# Site header

## Changes

This pull request makes the following changes:

- Sets `define( 'SHOW_GLOBAL_NAV', false );`
- Adds styling for social media icons, circled or not circled
- Enqueues dashicons
- Adds `font-body-display` font family
- Adds basic styling for body paragraphs
- Adds styling for desktop and mobile header

Desktop:
![Screen Shot 2019-11-22 at 1 12 08 PM](https://user-images.githubusercontent.com/18353636/69450021-30fe9680-0d2a-11ea-94b7-4107bb75bf77.png)

Mobile:
![Screen Shot 2019-11-22 at 1 12 16 PM](https://user-images.githubusercontent.com/18353636/69450020-30fe9680-0d2a-11ea-97fb-ef890f500e2f.png)

## Why



For #1

## Testing/Questions

Features that this PR affects:

- Site header, social media icons, global nav, etc. 


Questions that need to be answered before merging:

- [x] The borders in the mockup use `#CCC;` which is `@grey4`. Should we just remove this and use `#CDCDCD` which is `@grey3`?
- [ ] The social icons on the right side are just a list in a text widget instead of the Largo Follow widget icons. Is that ok, or should I rework it to use the Largo Follow widget instead?

Steps to test this PR:

1. Add the logo to theme images
![logo-tagline-stacked-color-srgb](https://user-images.githubusercontent.com/18353636/69450190-994d7800-0d2a-11ea-87ac-7bbeaf5b6339.png)
2. Theme Options -> Advanced -> Enable Optional Header Widget Area -> true
3. Add a text widget to the header widget area and add this content:
```
<label>&nbsp;</label>
<p>Acting in the public interest, the Caribbean Investigative Journalism Network is the region’s premier investigative journalism organisation producing engaging stories for multiple platforms</p>
```
4. Add a custom html widget to the header widget area and add this content:
```
<label>Start a Conversation</label>
<ul class="social-icons circled-icons">
 	<li><a href="#"><span class="dashicons dashicons-facebook-alt"></span></a></li>
 	<li><a href="#"><span class="dashicons dashicons-twitter"></span></a></li>
</ul>
```
